### ISSUE TITLE
Add parameter to Travis-CI configuration to specify which JDK(s) to build with

### ISSUE THREAD

**Issue by zednis** [konigio/konig#621]
Travis-CI allows for the specification of 1 or more JDK versions when building java projects.  For every specified JDK a new build will be launched.  This can be used to run builds against different versions of Java, different JDKs of the same version of Java, or simply as a means to specify/document the specific JDK we want to build with.

Test with only OpenJDK7
```
language: java
jdk:
- openjdk7
```

Test with OpenJDK7 and OracleJDK 7 & 8
```
language: java
jdk:
- openjdk7
- oraclejdk7
- oraclejdk8
```

**Comment by gmcfall** [comment]
If we are only supporting one JDK at a time, wouldn't it make more sense to specify the JDK in the parent maven POM so that all developers are using the same JDK as the CI build?

What's the advantage of declaring the JDK also in the Travis-CI config?

**Comment by zednis** [comment]
I don't believe there is a way to specify OpenJDK vs OracleJDK in the POM.  I think you can specify version, but not implementation.

This is done in Travis-CI so an open-source project that builds a tool that is intended to be compatible with both Java7 & 8 and both Oracle and OpenJDK implementations can test their build against those JDK implementations.

In a project at my prior employer we had a code project that worked in OracleJKD7 but not OpenJDK7 because some difference in their implementation broke one of our dependencies.
